selfdrive/test/test_routes.py

```diff
@@ -372,7 +372,7 @@
     'enableCamera': True,
     'enableDsu': False,
   },
-  "3fd5305f8b6ca765|2021-04-28--12-28-19": {
+  "3fd5305f8b6ca765|2021-04-28--19-26-49": {
```

Comment:
this one is also not uploaded

Reply:
Will this one work?
3fd5305f8b6ca765|2021-04-28--18-50-15

Reply:
I redid the test route, it was after a longer drive. I turned off the car before I started the test and on the useradmin it shows the 2 separate drives, on Cabana it combined them into one

Reply:
None of your routes are fully uploaded. make sure you have the upload raw toggle enabled in the settings.

Reply:
Ah, I didn't even think about that! Apologies, will it upload the previous drives after I enable the toggle, or do I need to run the test route again with the setting enabled?

Reply:
Looks like the drive should be uploaded now!